test/Libraries/RevitIntegrationTests/BugTests.cs

```diff
@@ -912,6 +912,36 @@ public void SelectedFaceIsTransformedCorrectly_MAGN_7977()
 
         [Test]
         [Category("RegressionTests")]
+        [TestModel(@".\emptyS.rvt")]
+        public void Rotation_MAGN_8056()
+        {
+            var model = ViewModel.Model;
+
+            string filePath = Path.Combine(workingDirectory, @".\Bugs\MAGN_8056.dyn");
+            string testPath = Path.GetFullPath(filePath);
+
+            ViewModel.OpenCommand.Execute(testPath);
+            AssertNoDummyNodes();
+
+            RunCurrentModel();
+
+            string locationNodeId = "8c9dbf40-73d7-4788-84b2-ccf015fa47de";
+            var locations = GetPreviewCollection(locationNodeId);
+            var point = locations[0] as Point;
+            point.X.ShouldBeApproximately(2.0);
+
+            string numberSliderNodeId = "971b7986-59e2-4ad1-b087-0e9e4158506a";
+            DoubleSlider slider = GetNode<DoubleSlider>(numberSliderNodeId) as DoubleSlider;
+            slider.Value = 10; // Set the new rotation
+
+            RunCurrentModel();
+            locations = GetPreviewCollection(locationNodeId);
+            point = locations[0] as Point;
+            point.X.ShouldBeApproximately(2.0);// The x coordination of the column should not change
```

Comment:
Can you also check for some value such as `Y` or `Z` coordinate that is supposed to change so that we know the rotation has happened?